source/energy/policy.rst

```diff
@@ -0,0 +1,37 @@
+.. policy:
+
+Modelling policies
+==================
+The global energy model distinguishes between eleven global regions (cf. Section :ref:`spatial`).  It is nevertheless important to represent current and planned national policies, such as the nationally determined contributions (NDCs) as agreed upon in Paris Agreement, at a lower geographical resolution in order to adequatly account for near term developments in scenario development processes. 
+
+Representation of single country Nationally Determined Contributions (NDCs)
+---------------------------------------------------------------------------
+The targets formulated in the NDCs come in many different flavors. This applies to the sectors and gasses, which policies cover, but also how these are expressed and quantified. In the global energy model, four broad categories of policy types related to the NDCs are represented, each of which is translated into a set of constraints.
+   1. Emission targets
+   2. Energy shares
+   3. Capacity or generation targets
+   4. Macro-economic targets
+A detailed description of the methodological implementation of the NDCs in the global energy model, along with an extensive list of the energy related targets considered can be found in Rogelj et al. (2017) :cite:`rogelj_indc_2017`. Additional policies implemented in the model can also be found in ('what reference for the CD_Links related policies?`)
+
+Emission targets
+----------------
+Country specific emission reduction targets are specified either in relation to a reference emission trajectory (in the form of a baseline or business as usual scenario (BAU)). The targets themselves are expressed as either (1) absolute reduction, (2) a percentage reduction or (3) intensity reductions e.g. emissions per GDP or per capita. In order to account for these different reduction targets in the global energy model, the targets are translated so that a regionally specific upper bound on emissions can be formulated. If not further specified, emission constraints are assumed to apply to all sectors and all gases, i.e. total GHGs.
```

Comment:
Please check first sentence: targets are specified either in relation to a reference emission trajectory (...), or [a base year reference?]

Reply:
@fabiosferra - updates made, please check if this is now clear?